website/pages/en/index.js

```diff
@@ -14,41 +14,41 @@ const siteConfig = require(process.cwd() + "/siteConfig.js");
 const translate = require("../../server/translate.js").translate;
 
 const pre = "```";
-const typeAnnotationExample= `${pre}hack
-<?hh
-class MyClass {
-  const int MyConst = 0;
-  private string $x = '';
-  public function increment(int $x): int {
-    $y = $x + 1;
-    return $y;
+const asyncExample= `${pre}hack
+  async function helloWorld(): Awaitable<string> {
+    return "Hello World"; // simple, trivial example
+  }
+
+  <<__EntryPoint>>
+  async function call_helloWorld(): Awaitable<void> {
+
+  $aw = helloWorld();
+  $result = await $aw; // Suspends until $aw completes
+  echo $result; // "Hello World"
   }
-}
 ${pre}`;
 const genericsExample = `${pre}hack
-<?hh
-class Box<T> {
-  protected T $data;
+  class Box<T> {
+    protected T $data;
 
-  public function __construct(T $data) {
-    $this->data = $data;
-  }
+    public function __construct(T $data) {
+      $this->data = $data;
+    }
 
-  public function getData(): T {
-    return $this->data;
+    public function getData(): T {
+      return $this->data;
+    }
   }
-}
 ${pre}`;
-const lambdaExample = `${pre}hack
-<?hh
-function foo(): (function(string): string) {
-  $x = 'bar';
-  return $y ==> $x . $y;
-}
-function test(): void {
-  $fn = foo();
-  echo $fn('baz'); // barbaz
-}
+const XHPExample = `${pre}hack
+  // Traditional: HTML elements require quotation marks
+  $user_name = 'Fred';
+  echo "<tt>Hello <strong>$user_name</strong></tt>";
```

Comment:
XHP guarantees that your tags are well-formed. Perhaps show how it's easy to screw up with strings?
```suggestion
  echo "<tt>Hello <strong>$user_name</tt></strong>";
```

XHP also protects you from XSS vulnerabilities, but that's a little more verbose :)